.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[alias]
```

Comment:
I currently have `.cargo/config.toml` symlinked to `.cargo/config_fast_builds`. This change will prevent doing so.

Reply:
@cart requested this approach [knowing it would cause this sort of conflict](https://github.com/bevyengine/bevy/pull/1309#discussion_r564023753).  That is to say, I'm acknowledging this problem as an accepted downside of this approach.

Reply:
Oh!  Though I should mention that since `.cargo/config.toml` is ignored by Git by default when looking at changes to include in commits, this should be straightforward to work around:

1. Delete your symlink
2. Now you can pull or switch to a branch that includes these changes
3. Then you can locally make whatever changes to `.cargo/config.toml` and it will be ignored.  Either re-establish your symlink, or copy the contents of `config_fast_builds` into it, or whatever you want.

Reply:
The disadvantage of that is that changes to config_fast_build are no longer automatically applied. In addition git rebase complains about changed files, thus requiring git stash and not forgetting git stash pop.

Reply:
Ya, that sounds annoying. 😦

Reply:
One solution should be to make a `.cargo/config.toml` symlinked to `config_fast_builds` but a level higher in the tree.

Reply:
@DJMcNab TIL! Nice find! [The docs say](https://doc.rust-lang.org/cargo/reference/config.html#hierarchical-structure) they'll merge `.cargo/config.toml` from _all_ parent directories.  So if I'm understanding correctly, if your `bevy` repo is located at `projects/bevy`, you can add a symlink:

* `projects/.cargo/config.toml -> ../bevy/.cargo/config_fast_builds`

...and the contents of that would be merged into whatever is in `bevy/.cargo/config.toml`.  It looks like the drawback of this approach is that any other rust project under `projects` will also be affected.

Reply:
Yeah, that's right

My project structure is:
```
repositories/
| priroda/
| <...>
| bevy/
 | .cargo/config.toml
 | my_bevy_game/
 | assistance/ (cargo workspace with one target directory for when I need to clone someone else's project)
 | bevy_mod_picking/
 | bevy/
  | .cargo/
   | config.toml
   | config_fast_builds.toml
```

Reply:
I'm starting to think that we just shouldn't use aliases. `cargo run --package ci` isn't that much harder to type than `cargo ci`. It is also clearer that it is a "package" in the workspace being run and not a "custom cargo command". And most importantly, it doesn't force us to stomp on config.toml (which is "user config").

Reply:
I'll make the change 👍🏼

Reply:
✅ `.cargo/config.toml` removed (removing the `cargo ci` alias, use `cargo run -p ci` or `cargo run --package ci` instead)